plugin/storage/cassandra/spanstore/dbmodel/tag_filter_whitelist.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// WhitelistTagFilter filters out all tags in its tags slice
+type WhitelistTagFilter struct {
```

Comment:
couldn't whitelist/blacklist filter be implemented with a single struct that tags a boolean parameter to change the behavior?

Reply:
Yup.  Combined into `ExactMatchTagFilter`.  In my mind it would be more efficient to have two different structs.  In retrospect that seems silly.